azure-pipelines.yml

```
@@ -193,6 +198,89 @@ stages:
           condition: always()
 
 - ${{ if eq(variables._RunAsPublic, True) }}:
+
+  - stage: Test_CLI_Package_In_Helix_Android
+    displayName: Run CLI End-to-End Helix scenarios - Android
+    dependsOn: Build_OSX
+    jobs:
+    - template: /eng/common/templates/jobs/jobs.yml
+      parameters:
+        workspace:
+          clean: all
+        jobs:          
+        - job: Linux
+          pool:
+            vmimage: ubuntu-latest
+          strategy:
+            matrix:
+              Build_Debug:
+                _BuildConfig: Debug
+          preSteps:
+          - checkout: self
+            clean: true
+          displayName: Submit Helix Job
+          steps:
+          - task: DownloadPipelineArtifact@2
+            inputs:
+              source: current
+              artifact: Microsoft.DotNet.XHarness.CLI.Debug
+              path: $(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)/Shipping/
+          
+          - script: eng/common/build.sh
+              --configuration $(_BuildConfig)
+              --prepareMachine
+              --ci
+              --restore
+              --test
+              --projects $(Build.SourcesDirectory)/tests/integration-tests/Android/Android.Helix.SDK.Tests.proj
+              /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/HelixRun.binlog
+              /p:RestoreUsingNuGetTargets=false
+            displayName: Run Helix Tests
+            env:
+              SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+              HelixAccessToken: ''
+
+  - stage: Test_CLI_Package_In_Helix_iOS
+    displayName: Run CLI End-to-End Helix scenarios - iOS
```

The other stages (and the directories) are all called "Integration tests". Maybe E2E is more fitting but I would just stick with one

```suggestion
    displayName: Run CLI Integration Helix scenarios - iOS
```